test/flow_control_SUITE.erl

```diff
@@ -23,7 +23,7 @@ init_per_testcase(
     PreChatterConfig =
         [
          {stream_callback_mod, server_connection_receive_window},
-         {initial_window_size, 64},
```

Comment:
This test had the initial_window_size for the streams way smaller than the default for the connection, so we weren't actually testing this case